static/js/components/EducationDisplay.js

```diff
@@ -113,14 +114,15 @@ export default class EducationDisplay extends ProfileFormFields {
           showLevelForm={true}
           validator={educationValidation}
         />
-        <Card shadow={1} className="profile-tab-card" id="education-card">
-          <Grid className="profile-tab-card-grid">
-            <Cell col={4} className="profile-card-title">
-              Education
+        <Card shadow={1} className="profile-form" id="education-card">
+          <Grid className="profile-form-grid">
+            <Cell col={12} className="profile-form-row profile-card-header">
+              <span>
```

Comment:
I think here the selector is actually `.profile-card-header > span`, should I still change it to something like `.profile-card-header > .title`?